security/MacPass/Portfile

```diff
@@ -1,45 +1,46 @@
-platforms               darwin
-license                 GPL-3
-maintainers             @janosch mailbox.org:janosch1 openmaintainer
-description             KeePass client for macOS
-long_description        ${name} is a native macOS port of KeePass. Key features are automatic \
-                        form filling and regex matching of window titles to detect the correct target application. \
-                        Through the plugin MacPassHTTP, ${name} can also be used together with \
-                        keepass browser extensions to integrate ${name} as a password database the browser can use.
-homepage                https://macpassapp.org/
+github.setup        MacPass MacPass ${macpass_hash}
+version             0.8-RC3
+categories          security
+platforms           darwin
+license             GPL-3
+maintainers         {@janosch mailbox.org:janosch1} openmaintainer
+description         KeePass client for macOS
+long_description    ${name} is a native macOS port of KeePass. Key features are automatic \
+                    form filling and regex matching of window titles to detect the correct target application. \
+                    Through the plugin MacPassHTTP, ${name} can also be used together with \
+                    keepass browser extensions to integrate ${name} as a password database the browser can use.
+homepage            https://macpassapp.org/
 
-master_sites            ${github.homepage}/archive:macpass \
-                        https://github.com/mstarke/DDHotKey/archive:ddhotkey
+master_sites        ${github.homepage}/archive:macpass \
+                    https://github.com/mstarke/DDHotKey/archive:ddhotkey
 
-distfiles               ${macpass_hash}.tar.gz:macpass \
-                        ${ddhotkey_hash}.tar.gz:ddhotkey
+distfiles           ${macpass_hash}.tar.gz:macpass \
+                    ${ddhotkey_hash}.tar.gz:ddhotkey
 
-checksums               ${macpass_hash}.tar.gz \
-                        rmd160  4002695949f0c846421e3dfd83443e6a33327495 \
-                        sha256  8757fe329a06d1798e40db2b617249bf2fcf66e112291e83867b329a2fd78e60 \
-                        size    11002547 \
-                        ${ddhotkey_hash}.tar.gz \
-                        rmd160  12ce0935b45c9b8a1e258f267ff8d18f0429eed0 \
-                        sha256  656d090cca863b236b7d319df4a5853b55ebf9af39a31a08cb471122c9bfbf56 \
-                        size    27575
+checksums           3cfbbb1f16437b2e1ae01ecde303ca1266960f8c.tar.gz \
+                    rmd160  61549b619d1dd0b4c0441004478594971ddfd4d8 \
+                    sha256  b70559e9b20316bf74cf6ea6bd2a13bf19d8c4abfac79e3056bda0264f97973f \
+                    size    11002723 \
+                    a996a0a3980de926168df7c2f8b02a1231760561.tar.gz \
+                    rmd160  12ce0935b45c9b8a1e258f267ff8d18f0429eed0 \
+                    sha256  656d090cca863b236b7d319df4a5853b55ebf9af39a31a08cb471122c9bfbf56 \
+                    size    27575
```

Comment:
since you defined the hash values already above I would probable keep that variable name here as you had it before

Reply:
You are right. I missed that while doing copy & paste.